shuffle/integration-tests/src/helper.rs

```diff
@@ -2,46 +2,67 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use anyhow::Result;
-use diem_sdk::{client::BlockingClient, types::LocalAccount};
-use shuffle::{account, deploy, new, shared::Home};
+use diem_sdk::{
+    client::BlockingClient, transaction_builder::TransactionFactory, types::LocalAccount,
+};
+use shuffle::{
+    account, deploy, new,
+    shared::{Home, NetworkHome},
+};
 use std::{path::PathBuf, str::FromStr};
 use tempfile::TempDir;
 use url::Url;
 
 pub struct ShuffleTestHelper {
     home: Home,
+    network_home: NetworkHome,
     tmp_dir: TempDir,
 }
 
 impl ShuffleTestHelper {
     pub fn new() -> Result<Self> {
         let tmp_dir = TempDir::new()?;
         let home = Home::new(tmp_dir.path())?;
-        Ok(Self { tmp_dir, home })
+        let network_home = NetworkHome::new(tmp_dir.path());
+        Ok(Self {
+            tmp_dir,
+            home,
+            network_home,
+        })
     }
 
     pub fn home(&self) -> &Home {
         &self.home
     }
 
+    pub fn network_home(&self) -> &NetworkHome {
+        &self.network_home
+    }
+
     pub fn project_path(&self) -> PathBuf {
         self.tmp_dir.path().join("project")
     }
 
     pub fn create_accounts(
         &self,
         treasury_account: &mut LocalAccount,
+        new_account: LocalAccount,
+        factory: TransactionFactory,
         client: BlockingClient,
     ) -> Result<()> {
-        account::create_local_accounts(&self.home, client, treasury_account)
+        account::create_account_onchain(treasury_account, &new_account, &factory, &client)
```

Comment:
we're obviously creating an account onchain. Can we revert back to the old name or remove onchain so it's not redundant.